source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -492,6 +525,32 @@ void mgmtNoKeyRequest::onallChildRespAgrregate(Common::Redis::RespValuePtr&& val
       bool positiveresponse = true;
       updateStats(error_count_ == 0);
       if (!pending_responses_.empty()) {
+        if ( rediscommand == "info"){
```

Comment:
Good to have: We can have support for subcommand for info like info keyspace etc,.